drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -31,6 +31,9 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   has_many :current_living_situations, through: :enrollments
   has_many :hmis_services, through: :enrollments # All services (HUD and Custom)
 
+  has_one :warehouse_client_source, class_name: 'GrdaWarehouse::WarehouseClient', foreign_key: :source_id, inverse_of: :source
+  has_one :destination_client, through: :warehouse_client_source, source: :destination, inverse_of: :source_clients
```

Comment:
Hmm, I think this should be somewhat temporary.  We'd like to have a warehouse API which would be a more intentional way of accessing warehouse related data.  This feels a bit like a back-slide into that data which might end up circumventing access controls.